index.html

```diff
@@ -1680,6 +1680,13 @@ <h4 id="signed-raw">Signed Raw Transaction</h4>
       // Select and return UTXO pointers (filters applied)
       const cCoinControl = { nValue: 0, nChange: 0, arrSelectedUTXOs: [] }
       for (const cUTXO of arrUTXOs) {
+
+        //we dont want the MN collateral to be spent
+        if(localStorage.getItem("masternode")){
+          if(JSON.parse(localStorage.getItem("masternode")).collateralTxId === cUTXO.id){
+            continue
```

Comment:
Are we removing the tx when we create the masternode as well?

Reply:
Yep, added in https://github.com/PIVX-Labs/MyPIVXWallet/pull/49/commits/70e973790bf3408e4f7fa4ec42624a51eb20ee97

Reply:
The collateral utxo is locked only when you successfully access the masternode (enable or pre enabled)